docs/reference/kubectl-plugin.md

```diff
@@ -98,7 +98,12 @@ Resource declaratively, as a yaml.
     #        - "true"
     ```
 
-1. Install cluster
+   and install a cluster
     ```bash
     kubectl storageos install --stos-cluster-yaml StorageOSCluster.yaml --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
```

Comment:
This is not part of the change, but we are here, so I'd compact the shell lines for readability

```
kubectl storageos install \
    --stos-cluster-yaml StorageOSCluster.yaml \
    --etcd-endpoints "storageos-etcd-client.storageos-etcd:2379"
```